src/Schedulers/ImportScheduler.php

```diff
@@ -38,8 +38,20 @@ public static function is_importing() {
 				'group'    => self::$group,
 			)
 		);
+		if ( empty( $pending_jobs ) ) {
+			$in_progress = self::queue()->search(
+				array(
+					'status'   => 'in-progress',
+					'per_page' => 1,
+					'search'   => 'import',
+					'group'    => self::$group,
+				)
+			);
+		} else {
+			$in_progress = true;
+		}
 
-		return ! empty( $pending_jobs );
+		return ! empty( $in_progress );
```

Comment:
The logic with the if/else above was a little hard to follow.  Maybe we could make this:

```php
return ! empty( $pending_jobs ) || ! empty( $in_progress_jobs );
```

Reply:
Not sure if you've finished updating or not, but I was thinking we remove the latter else portion:

```
else {
	$in_progress = true;
}
```

In favor of checking `$pending_jobs` and `$in_progress_jobs`.

Reply:
@joshuatf Thanks for the followup. Removed in `0d42422`.